### DISCUSSION TITLE
New version

### DISCUSSION THREAD

**Discussion by pdifeo** [rustdesk/rustdesk-server-pro#84] [Q&A] [answered]
I have seen that version 1.1.10 has been released. is it possible to upgrade and continue using version 1.2.2 of the client?

Pasquale

**Comment by dinger1986** [comment]
I still have clients on 1.2.2 and works fine

**Reply by pdifeo** [reply]
Is there a release note on features and bugs fixed ?

**Reply by dinger1986** [reply, marked as answer]
https://github.com/rustdesk/rustdesk-server-pro/releases/tag/1.1.10

Click on Chnagelog
